backend/database/migrations/2023_06_09_025950_create_transactions_table.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateTransactionsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('transactions', function (Blueprint $table) {
+            $table->id();
+            $table->unsignedBigInteger('account_id');
+            $table->foreign('account_id')->references('id')->on('accounts')->constrained()->onDelete('cascade');
+            $table->dateTime('transaction_date');
+            $table->enum('transaction_type', [0, 1, 2, 3]);
+            $table->enum('category',[0, 1, 2, 3, 4, 5, 6]);
```

Comment:
are there description which value means which? for example:

transaction_type = 0 means deposit